client/components/themes-list/docs/example.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThemesList from 'components/themes-list';
+
+const onScreenshotClickFn = function( theme, index ) {
+	console.log( `Theme ${ theme.id } at ${ index } clicked` );
+};
+
+const ThemesListExample = React.createClass( {
+	displayName: 'ThemesList',
```

Comment:
This seems like improper use of `displayName`, as our component here just wraps around the real `ThemesList`, so no need to disguise it. And since the previous line immediately binds the component to `const ThemesListExample`, React might be smart enough not to need `displayName` at all.

So I'd either remove the property or rename its value to `ThemesListExample`.